### ISSUE TITLE
>>if >>else >>end-if : "" unexpected tokens ignored

### ISSUE THREAD

**Issue by FALLAI-Denis** [IBM/zopeneditor-about#65]
## Development environment used

- Editor Platform
  - [X] Visual Studio Code
  - [ ] CodeReady Workspaces
  - [ ] Eclipse Che
  - [ ] Standlone Theia
- Platform Version: VS Code 1.46.1 / ZOE 1.0.0
- Operating System: Windows 10 PRO 64 "1607"
- Java Version: 1.8
- Logs attached: no

## Problem Description

Detailed steps for reproducing the problem:

create source code with **>>if** text **>>else** text **>>end-if**

### Observed behavior

the text between **>>else** and **>>end-if** appears in error, but the error is not clearly indicated.
Commenting out **>>else** removes the error.

### Expected behavior

No error ;-)

![if-else-endif-01](https://user-images.githubusercontent.com/63861965/85874565-1d8d3f80-b7d3-11ea-887c-db54b27c48c4.png)

![if-else-endif-02](https://user-images.githubusercontent.com/63861965/85874667-47466680-b7d3-11ea-8ecd-5da1e0615478.png)

![if-else-endif-05](https://user-images.githubusercontent.com/63861965/85876567-33e8ca80-b7d6-11ea-9e75-df8f4a8fded2.png)

![if-else-endif-03](https://user-images.githubusercontent.com/63861965/85875749-fd5e8000-b7d4-11ea-8c54-851bf1f75040.png)

![if-else-endif-04](https://user-images.githubusercontent.com/63861965/85875756-018a9d80-b7d5-11ea-896c-6afc81884f79.png)

![if-else-endif-06](https://user-images.githubusercontent.com/63861965/85876820-8cb86300-b7d6-11ea-8566-1b9d52d86e6b.png)

**Comment by gplambert** [comment]
@FALLAI-Denis - can you include a screen shot of your settings?  Particularly your "files.associations" and "cobolDatasets"?

**Comment by FALLAI-Denis** [comment]
@gplambert

```
    "files.associations": {
        "*.COBOL*": "cobol",
        "*.COB*": "cobol",
        "*.COBCOPY*": "cobol",
        "*.COPYBOOK*": "cobol",
        "*.COPY*": "cobol",
        "*.PL1*": "pl1",
        "*.PLI*": "pl1",
        "*.INC*": "pl1",
        "*.INCLUDE*": "pl1",
        "*.JCL*": "jcl",
        "*.ASM*": "hlasm",
        "*.jcl": "jcl",
        "*.cbl": "cobol",
        "*.cpy": "cobol",
        "*.asm": "hlasm"
    },
```
```
    "zopeneditor.datasets": {
        "hlasmDatasets": [".ASM"],
        "cobolDatasets": [
          ".COBOL",
          ".COB",
          ".COBCOPY",
          ".COPYBOOK",
          ".COPY"
        ],

```

**Comment by gplambert** [comment]
@FALLAI-Denis  Thanks, I will put this issue on our bug fix list.

Do you happen to use IDz?  If so, have you tried these scenarios?

**Comment by FALLAI-Denis** [comment]
@gplambert We don't use IDz.

**Comment by phaumer** [comment]
Fixed in Z Open Editor 1.1.
